packages/vuetify/src/components/VHover/__tests__/VHover.spec.cy.tsx

```diff
@@ -0,0 +1,62 @@
+/// <reference types="../../../../types/cypress" />
+
+import { CenteredGrid } from '@/../cypress/templates'
+import VHover from '../VHover'
+
+describe('VHover', () => {
+  it('should react on mouse events', () => {
+    cy.mount(() => (
+      <CenteredGrid width="200px" class="v-theme--light">
```

Comment:
`width="200px" class="v-theme--light"`

is this something that should be put into centered grid by default?